### ISSUE TITLE
Error getting serveral query parameters in one handler

### ISSUE THREAD

**Issue by lambdaburrito** [nickel-org/nickel.rs#186]
I'm not sure if this is an API bug or I am just misusing it but I cannot compile when I am retrieving several query parameters in one handler.

```rust
router.get("/query", middleware! { |request|
    let x = request.query("x", "test");
    let y = request.query("y", "test");
    let z = request.query("z", "test");
    format!("Query params are: {:?},{:?},{:?}", x, y, z)
});
```
It complains that I cannot borrow more than once from a *req.

**Comment by Ryman** [comment]
Nice spot, this might be tricky to resolve, but I'll look into it soon!

**Comment by lambdaburrito** [comment]
Cheer Ryman! Good luck

**Comment by Ryman** [comment]
@lambdaburrito #195 should deal with this

**Comment by Ryman** [comment]
Fixed in #195
